components/brave_wallet/browser/keyring_service.h

```diff
@@ -240,6 +242,7 @@ class KeyringService : public KeyedService, public mojom::KeyringService {
   FRIEND_TEST_ALL_PREFIXES(KeyringServiceUnitTest, UnlockResumesDefaultKeyring);
   FRIEND_TEST_ALL_PREFIXES(KeyringServiceUnitTest,
                            GetMnemonicForDefaultKeyring);
+  FRIEND_TEST_ALL_PREFIXES(KeyringServiceUnitTest, ValidatePassword);
```

Comment:
Why do we need to friend the test case here? I don’t see it accesses any private members

Reply:
Not needed
Good catch, thank you!